roles/telemetry_logging/tasks/main.yml

```diff
@@ -3,3 +3,14 @@
   when: journal_list is defined
   ansible.builtin.include_tasks: "journal_tests.yml"
   loop: "{{ journal_list }}"
+
+#All containers need to be up in order for this task to pass
+- name: "[TEST] verify status of multiple containers"
+  ansible.builtin.include_role:
+    name: common
+  vars:
+      common_container_test_id: "RHOSO-12753"
+      common_container_list:
+          - ceilometer_agent_compute
+          - ceilometer_agent_ipmi
+          - node_exporter
```

Comment:
I don't see any connection of these containers to logging. ceilometer_agent_compute and node_exporter are deployed by the "telemetry" service. ceilometer_agent_ipmi will be deployed by the "telemetry power monitoring" service. You can deploy logging on the dataplane without these services. That means, that logging would work correctly, but these containers won't be there. If we want, we could check these containers in some general compute telemetry role, but not in a logging role. What we could check for logging on the computes is that rsyslog is installed and running, but I think rsyslog runs directly on the host, not inside a container.